content/docs/for-app-developers/how-to/build-outputs/understand-failures.md

```diff
@@ -4,6 +4,6 @@ weight=99
 summary="How to troubleshoot when things go wrong."
 +++
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+While `Buildpacks` help developers transform applications' source code into container images that can run on any cloud, creating an error-free experience remains far from achieved.
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+This guide explores some of the most common issues that may prevent image build completion and provides troubleshooting tips to help end-users navigate these issues.
```

Comment:
Let's drop this bit and leave it as a stub.  I think we need to trawl back though the issues on https://github.com/buildpacks/pack to figure out the FAQs.

Reply:
@natalieparellano still waiting on this

Reply:
Please see https://github.com/buildpacks/docs/pull/777#discussion_r1841099965 - includes an example and a template for us or others in the community to add more.

If you don't like it though, I'm fine with changing this page back into a stub.